source/_patterns/03-organisms/sections/timeline/timeline.yml

```diff
@@ -3,18 +3,24 @@ lead: Nullam at semper mauris, at rhoncus purus.
 timeline:
   - title: Create a settlement strategy
     excerpt: "<p>Before you enter into a negotiation, you should create a strategy as to how you will enter into negotiations.</p>"
-    read_more_link: "Strategy Details"
+    read_more:
+      label: "Strategy Details"
+      url: "#"
   - title: "Decide how to negotiate"
     excerpt: "<p>Decide how you wish to negotiate. Common choices are verbally, usually by phone, or in writing.</p>"
-    read_more_link: "Negotiation Details"
+    read_more:
+      label: "Negotiation Details 2"
+      id: "jcc-full-modal"
```

Comment:
See comment above.  You use this same id in the `full-modal.yml` so it would become a duplicate id accessibility error. Changing it to `timeline-full-modal-2` would be better.